src/app/organizations/organization/organization.component.html

```diff
@@ -60,13 +65,14 @@ <h1 [matTooltip]="org?.name">{{org.displayName}}</h1>
           </span>
           <span fxLayout="row" fxLayoutGap="10px" *ngIf="org?.link">
             <mat-icon>link</mat-icon>
-            <span><a [href]="org.link" target="_blank">{{ org.link }}</a></span>
+            <span><a [href]="org.link" target="_blank">{{ org?.link }}</a></span>
```

Comment:
You should add `rel="noopener"`. See [this](https://developers.google.com/web/tools/lighthouse/audits/noopener)